src/pages/message/signupSuccess/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { NextPage } from "next";
+
+const SignupSuccess: NextPage = () => {
+  return (
+    <>
+      <div>
```

Comment:
We can add `Head` to define the title and metadata too.

Reply:
`@next/head` can only be used to define custom titles, but it's been fixed.